connectors/sources/graphql_queries.py

```diff
@@ -0,0 +1,322 @@
+from enum import Enum
+
+
+class GithubQuery(Enum):
```

Comment:
suggestion: As this is specific to GitHub I would keep it in `github.py`. I don't expect this queries to be reused across different modules, but to be only used inside the `github.py` module.

Reply:
I don't have any concern that the `github.py` module contains a high number of LOC. If we've more connectors using GraphQL it's likely that the `graphql_queries.py` will be huge and contain code from many unrelated data sources/connectors. Keeping high cohesion in mind I think it's fair to say, that github specific code should live inside the same module, if there's no need to share the code.

Reply:
okay, moved queries in `github.py` file